README.md

```diff
@@ -1,2 +1,5 @@
-# tab-reloader
-Auto-reloader of Chrome tabs
+# tab-reloader: Auto-reloader of Chrome tabs
+
+Have you ever sat there refreshing a page every few minutes to see if your package had any shipping updates, or to see if pre-orders had opened, or to see if you had any new messages? NO LONGER!
+
+This is a Chrome extension that periodically refreshes tabs at an interval, so you don't have to. In the future, I hope to add functionality that checks to see if something on the page has changed and creates a notification if it does, so that not only do you not have to keep manually pressing the refresh button - you also don't have to sit and watch the browser window.
```

Comment:
Can you include information about how to install this extension?

Reply:
Great idea, @srambach. Thanks for the suggestion!